tests/output.rs

```diff
@@ -67,9 +66,35 @@ fn strip_indent<'line>(line: &'line str, indent: &str) -> &'line str {
     &line[indent.len()..]
 }
 
+#[cfg(target_os = "macos")]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".dylib";
+#[cfg(all(unix, not(target_os = "macos")))]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".so";
+#[cfg(windows)]
+const DYNAMIC_LIBRARY_EXTENSION: &str = ".dll";
+
+#[cfg(unix)]
+fn get_dynamic_library_name(library_name: &str) -> String {
+    format!("lib{library_name}{}", DYNAMIC_LIBRARY_EXTENSION)
+}
+#[cfg(windows)]
+fn get_dynamic_library_name(library_name: &str) -> String {
+    format!("{library_name}{}", DYNAMIC_LIBRARY_EXTENSION)
+}
```

Comment:
And same here, I think it'd be a little cleaner to write an `if cfg!` inside the function rather than repeating the function definition.